lib/bibliothecary/parsers/maven.rb

```diff
@@ -152,9 +152,12 @@ def self.parse_maven_resolved(file_contents)
           .compact
           .uniq
       end
+
       def self.parse_maven_tree(file_contents)
         file_contents = file_contents.gsub(/\r\n?/, "\n")
         captures = file_contents.scan(/^\[INFO\](?:(?:\+-)|\||(?:\\-)|\s)+((?:[\w\.-]+:)+[\w\.\-${}]+)/).flatten.uniq
+        captures.shift if captures.size > 1 # first dep line will be the package itself (unless we're only analyzing a single line)
```

Comment:
The `size > 1` constraint is because I've seen this method used to parse a single line.